src/chainparams.cpp

```diff
@@ -262,10 +262,10 @@ class CMainParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_DIP0003].nThreshold = 3226; // 80% of 4032
 
         // The best chain should have at least this much work.
-        consensus.nMinimumChainWork = uint256S("0x00000000000000000000000000000000000000000000112df08fc9f0ca551d0d"); // 1033120
+        consensus.nMinimumChainWork = uint256S("00000000000000000000000000000000000000000000146103ebd818111fae85"); // 1067570
```

Comment:
The removal of the `0x` prefix stood out to me. Any reason it was removed here?

Reply:
Good catch! 👍 I always do the same mistake tbh 🙈 Not sure why `0x` is here in the first place though, it's not used in any meaningful way https://github.com/dashpay/dash/blob/master/src/uint256.cpp#L38-L40... but this PR is definitely not a good place to change the format 🙂

Reply:
:man_facepalming: